lms/djangoapps/teams/utils.py

```diff
@@ -16,3 +17,23 @@ def emit_team_event(event_name, course_key, event_data):
 
     with tracker.get_tracker().context(event_name, context):
         tracker.emit(event_name, event_data)
+
+
+def are_team_submissions_enabled(course_key):
+    """
+    Checks to see if the CourseWaffleFlag or Django setting for team submissions is enabled
+
+    Returns:
+        Boolean value representing switch status
+    """
+    waffle_namespace = 'openresponseassessment'
+    switch_name = 'team_submissions'
+
+    if CourseWaffleFlag(waffle_namespace, switch_name).is_enabled(course_key):
+        return True
+
+    feature_name = 'ENABLE_ORA_TEAM_SUBMISSIONS'
```

Comment:
Shouldn't this be the first check - to prevent displaying team submissions if ORA team submissions are enabled regardless of the waffle flag?

Reply:
This allows a course override to take precedence, otherwise it's a binary on/off per environment.

Reply:
But what happens if we want the team submissions feature to be completely off - this logic will "overrule" the global setting based on a waffle flag. I think we should return true only if the waffle is on and the feature is on. If the feature is off, we should return false.